packages/core/src/shared/vscode/setContext.ts

```diff
@@ -37,6 +37,7 @@ export type contextKey =
     | 'gumby.wasQCodeTransformationUsed'
     | 'amazonq.inline.codelensShortcutEnabled'
     | 'aws.toolkit.lambda.walkthroughSelected'
+    | 'aws.amazonq.isSagemakerUnifiedStudio'
```

Comment:
put this next to `aws.isSageMaker` . helps discovery.

```suggestion
    | 'aws.isSagemakerUnifiedStudio'
```

Reply:
Thanks, addressing in next revision